.gitignore

```diff
@@ -2,3 +2,4 @@
 composer.lock
 phpunit.xml
 vendor
+.idea
```

Comment:
Should be in your own global .gitignore file, best practice because not part of this package